src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2295,64 +2295,6 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
             sGameEventMgr->StartEvent(eventId, true);
             break;
         }
-        case SMART_ACTION_START_CLOSEST_WAYPOINT:
```

Comment:
I'm not using it myself, but the action type is probably used in some private projects (I suppose). What it's good for is when a mob is summoned at a random location and there's a different waypoint per summon location. So this way it can just always start the one that is nearest by its summon location.

Besides, deleting a working SAI event or action type is kind of.. silly. To me, at least.